plugins/agentic-docs/skills/platform/templates/domain-concept-template.md

```
@@ -0,0 +1,74 @@
+```
+**Use case**: When to use this pattern
+
+### Pattern 2
+```yaml
+# Another example
+```
+**Use case**: When to use this pattern
+
+## Lifecycle / Workflow
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)